release-tool

```diff
@@ -37,9 +37,10 @@ DOCKER_CONTAINER_NAME="keepassxc-build-container"
 CMAKE_OPTIONS=""
 COMPILER="g++"
 MAKE_OPTIONS="-j8"
-BUILD_PLUGINS="autotype http yubikey"
+BUILD_PLUGINS="autotype http yubikey sshagent browser"
```

Comment:
We need to give users time to transition over. Perhaps we issue a warning banner on startup if they have http enabled to move to browser.

Reply:
I agree. But not for more than a few versions. Removing this here now will eliminate the need for another PR when we finally remove it.

Reply:
If we remove this here and use up-to-date release-tool to build 2.3.0 we will ship it without HTTP.
I think we should keep this at least 2-3 month and make an announcement with 2.3.0 that it will be deprecated soon

Reply:
This variable can be overridden from the command line.

Reply:
Corrected in last push